package.json

```diff
@@ -96,6 +96,7 @@
     "mysql": "^2.18.1",
     "mysql2": "^2.1.0",
     "nyc": "^15.0.0",
+    "oracledb": "^4.2.0",
```

Comment:
Considering that testing Oracle is the slowest step, do we want to run it on all Node versions?

Reply:
We could add an additional fast test that doesn't pull oracle, but I think its better to have more complete tests than fast ones. In either way it will take long to all test setups to finnish

Reply:
Yes, because in other case we can miss some breaking changes from node for oracledb